test/types.jl

```diff
@@ -379,3 +379,7 @@ end
 # Issue #52
 dt = now()
 @test_throws ErrorException ZonedDateTime(dt, warsaw) > dt
+
+# type extrema
+@test typemin(ZonedDateTime) <= ZonedDateTime(typemin(DateTime), utc)
+@test typemax(ZonedDateTime) >= ZonedDateTime(typemax(DateTime), utc)
```

Comment:
Won't these always  be `==`?

Reply:
Given the current definition, yes. But you could conceivably change the definition to restrict the interval to when time zones exist, for example.